### PR TITLE
Create talal.md

### PR DESCRIPTION
**Pull request by talalmahmood**

### CODE REVIEW

File: profiles/talal.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Talal Chaudhry
+
+![](https://github.com/talalmahmood.png)
+
+## Sports
+- Football
+- Cricker
```
Comment (elezar) [inline profiles/talal.md]: Spelling mistake. Should be "Cricket".

**Comment by talalmahmood** [comment]
Yes :)

File: profiles/talal.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# Talal Chaudhry
+
+![](https://github.com/talalmahmood.png)
+
+## Sports
+- Football
+- Cricket
```
Comment (brntbeer) [inline profiles/talal.md:7]: I finally learned how cricket (mostly) works! 💯

**Comment by brntbeer** [comment]
@talalmahmood looks good to me! 👍 :shipit: 
I still don't understand why I can change all comments here... ❓

**Pull request merged**